config/custom_schema_types.config

```diff
@@ -120,6 +155,35 @@ custom_schema_types {
         }
     }
 
+    /**
+    * Check if proper VCF entry list
+    */
+    check_vcf_list = { Map options, String name, Map properties ->
+        custom_schema_types.check_if_list(options[name], name)
+        for (item in options[name]) {
+            custom_schema_types.check_if_namespace(item, name)
+            properties.elements.each { key, val ->
+                schema.validate_parameter(item, key, val)
+            }
+        }
+    }
+
+    /**
+    * Check that at least one kind of input is given and only one of BAM or VCF is given
+    */
+    check_input_presence_and_exclusivity = { Map options ->
+        def bam_given = options.containsKey('bam')
+        def vcf_given = options.containsKey('vcf')
+
+        if (!bam_given && !vcf_given) {
+            throw new Exception("At least one input type (BAM or VCF) must be provided.")
+        }
+
+        if (bam_given && vcf_given) {
+            throw new Exception("Only one input type (either BAM or VCF) should be provided.")
+        }
+    }
+
```

Comment:
Yes, `check_input_presence_and_exclusivity` was accidentally left behind.

I guess `check_vcf_list` can't be used and isn't necessary because of the way the VCFs are input.  I removed the function and the VCFEntryList that referred to it.